__tests__/app.test.js

```diff
@@ -94,27 +93,71 @@ describe("/api/articles/:article_id", () => {
   });
 });
 
-  describe("/api/articles", () => {
-    it("GET responds with status code 200", () => {
-      return request(app).get("/api/articles").expect(200);
-    });
-    it("GET responds with slug and description fields", () => {
-      return request(app)
-        .get("/api/articles")
-        .then((response) => {
-            expect(response.body.articles).toBeSortedBy("created_at", { descending: true });
-            expect(response.body.articles.length).toBe(13)
-            response.body.articles.forEach((article) => {
-            expect(article).toMatchObject({
-                author: expect.any(String),
-                title: expect.any(String),
-                article_id: expect.any(Number),
-                topic: expect.any(String),
-                created_at: expect.any(String),
-                votes: expect.any(Number),
-                article_img_url: expect.any(String),
-                comment_count: expect.any(Number)
-              });
+describe("/api/articles", () => {
+  it("GET responds with status code 200", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  it("GET responds with slug and description fields", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((response) => {
+        expect(response.body.articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        expect(response.body.articles.length).toBe(13);
+        response.body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+      });
+  });
+});
+
+describe("/api/articles/:article_id/comments", () => {
+  it("GET responds with status code 200", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        console.log(response.body.comments)
+        expect(response.body.comments).toBeSortedBy("created_at", {
+          descending: true,
+        })
+        expect(response.body.comments.length).toBe(11);
+        expect(response.body.comments[0].article_id).toBe(1);
```

Comment:
You want this line to be true of every single comment, not just the first one - move this down into your forEach loop s out applies to each comment